# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/telemetry/Histograms.json
# Commit: 3b150d676b31
# Full Hash: 3b150d676b3122bffa703813876d1fbc35e10d17
# Author: Randell Jesup <rjesup@mozilla.com>
# Date: 2024-03-19 16:41:28
# Regressor Bug: 1356686
# File Overlap Count: 1
# Description:
#   Bug 1356686 - Do decompression off main thread in content r=necko-reviewers,valentin,extension-reviewers,robwu
#   
#   Content processes will now always retarget delivery of OnDataAvailable for Http
#   channels off the main thread. Consumers that were previously redirecting
#   off-main thread are not affected and their retargeting will stick, but any
# ==============================================================================

diff -r 54754a50d992 -r 3b150d676b31 toolkit/components/telemetry/Histograms.json
--- a/toolkit/components/telemetry/Histograms.json	Mon Mar 18 20:04:00 2024 +0000
+++ b/toolkit/components/telemetry/Histograms.json	Mon Mar 18 20:04:01 2024 +0000
@@ -3643,7 +3643,7 @@
     "releaseChannelCollection": "opt-out",
     "description": "Recorded once for each HTTP 401 response. The value records the type of authentication and the TLS-enabled status. (0=basic/clear, 1=basic/tls, 2=digest/clear, 3=digest/tls, 4=ntlm/clear, 5=ntlm/tls, 6=negotiate/clear, 7=negotiate/tls)"
   },
-  "HTTP_CHILD_OMT_STATS": {
+  "HTTP_CHILD_OMT_STATS_2": {
     "record_in_processes": ["content"],
     "products": ["firefox", "fennec"],
     "alert_emails": ["necko@mozilla.com"],
@@ -3657,7 +3657,8 @@
       "successMainThread",
       "failListener",
       "failListenerChain",
-      "notRequested"
+      "notRequested",
+      "successOnlyDecomp"
     ]
   },
   "TLS_EARLY_DATA_NEGOTIATED": {
